catalogd/internal/storage/localdir_test.go

```diff
@@ -540,12 +592,46 @@ entries:
 	testPackageName := "webhook_operator_test"
 	testChannelName := "preview_test"
 
+	testMultiPackageName := "multi_operator_test"
+	testMultiChannelName := "review_test"
+	testMultiBundleName := "multi-bundle.v0.0.1"
+
 	testPackage := fmt.Sprintf(testPackageTemplate, testPackageDefaultChannel, testPackageName)
 	testBundle := fmt.Sprintf(testBundleTemplate, testBundleImage, testBundleName, testPackageName, testBundleRelatedImageName, testBundleRelatedImageImage, testBundleObjectData)
 	testChannel := fmt.Sprintf(testChannelTemplate, testPackageName, testChannelName, testBundleName)
+	testMultiChannel := fmt.Sprintf(testChannelTemplate, testMultiPackageName, testMultiChannelName, testMultiBundleName)
```

Comment:
The test I'm adding in my PR queries with `package` so we don't have to add this extra channel here.